pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -477,3 +478,23 @@ func (t *FSTree) SetCompressor(cc *compression.Config) {
 func (t *FSTree) SetReportErrorFunc(_ func(string, error)) {
 	// Do nothing, FSTree can encounter only one error which is returned.
 }
+
+// CleanUpTmp removes all temporary files garbage.
+func (t *FSTree) CleanUpTmp() error {
```

Comment:
This in fact should fail in RO.

Reply:
Looks like it's still not RO-protected.

Reply:
Sorry, I didn't understand it the first time, but now I've got it.